Assignment9/Assignment9.Tests/MainWindowViewModelTests.cs

```diff
@@ -0,0 +1,143 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+
+namespace Assignment9.Tests
+{
+    [TestClass]
+    public class MainWindowViewModelTests
+    {
+        [TestMethod]
+        public void Contacts_ReturnsObservableContactCollection()
+        {
+            var viewModel = new MainWindowViewModel();
+
+            Assert.AreEqual(typeof(ObservableCollection<Contact>), viewModel.Contacts.GetType());
```

Comment:
A better way to do type checking is by simply using the `is` keyword. Checks with `.GetType()` quickly get problematic when you start dealing with derived types.
```suggestion
            Assert.IsTrue(viewModel.Contacts is ObservableCollection<Contact>);
```